### DISCUSSION TITLE
Ability to control the A/T with your throttle

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#723] [Ideas]
This is an option in the felis 742 and is always available in the zibo. What I mean by this, is when you move your physical joystick throttle, it will move the throttle lever in the aircraft. This is of course representing real life, but should be an option in the EFB as it can sometimes get annoying if you knock your throttle at home and it messes it all up in sim. This would be particularly useful when landing as the magknight doesn’t simulate automatically going idle on the throttle in the flare.

**Comment by aeroniemi** [comment]
The real plane doesn't automatically idle the engines on flare - that has to be done manually by the flight crew (and would be a bit dangerous if it did do that) - we have a *disconnect A/T at idle* option in the EFB that i think covers this feature request

**Reply by Simleybob** [reply]
It would also be useful in an auto land scenario, when the pilot it making changes constantly on the throttle. But alright for the rest.

**Reply by Simleybob** [reply]
Btw, when landing on all videos of the 787 in cockpit, I have watched they have been controlling the throttle by hand when landing, but still on a/t. Obviously you cannot do this in the magknight at the moment. E.g. https://youtu.be/Adse-ytbuzA

**Reply by aeroniemi** [reply]
Doing that doesn't really work in the sim - it requires your throttles to be synced (ie motorized) to meet the aircraft's idea of where the throttles are, and that just isn't possible without specialist hardware - if we did let you do this it'd cause more problems as any throttle movement would lead to an immediate jump in position, which i think would be counterproductive.
